tibanna_ffcommon/portal_utils.py

```diff
@@ -74,6 +87,14 @@
 import pkg_resources
 logger.debug(pkg_resources.get_distribution('dcicutils').version)
 
+OUTPUT_ARG_TYPE_LIST = [OUTPUT_PROCESSED_FILE,
+                        OUTPUT_REPORT_FILE,
+                        OUTPUT_QC_FILE,
+                        GENERIC_QC_FILE,
+                        OUTPUT_TO_BE_EXTRA_INPUT_FILE]
+
+# File types that are uploaded to the portal (and S3)
+PROCESSED_FILE_TYPES = [OUTPUT_PROCESSED_FILE, GENERIC_QC_FILE]
```

Comment:
I noticed this looking at the portal - is the implication here that there are two metadata records for QC files? One processed file and one QC object?

Reply:
Yes, there is one QC file (processed file) and the QualityMetricsGeneric object that contains the data of that QC file and a link to the metrics report zip file.